disneyland/server.go

```diff
@@ -24,30 +24,26 @@ func (s *Server) Init() {
 
 func (s *Server) CreateJob(ctx context.Context, in *Job) (*Job, error) {
 	user := getAuthUserFromContext(ctx)
-	in.Project = user.Project
-
-	created_job, err := s.Storage.CreateJob(in, user)
-	if err != nil {
-		return nil, detailedInternalError(err)
+	if in.Project == "" {
+		in.Project = user.Project
```

Comment:
Job should be forced to user's project, otherwise user would be able to write to other projects.